frontend/packages/client/src/components/ProposalInformation/Information.js

```diff
@@ -57,25 +58,38 @@ export default function Information({
         title="Voting strategy"
         component={
           <div className="is-flex" onClick={openStrategyModal}>
-            <div className="cursor-pointer has-text-grey">{strategyName}</div>
+            <div className="cursor-pointer has-text-grey">
+              {customStrategy ? customStrategy.name : strategyName}
+            </div>
```

Comment:
<img width="306" alt="Screen Shot 2022-10-12 at 12 32 03" src="https://user-images.githubusercontent.com/7526740/195385688-dcdc0625-8c59-464c-b7f9-29b8445ff4fc.png">

also we might not need to allow it it be clickable and to open the modal:

```
component={
          <>
            {customStrategy ? (
              <div className="is-flex has-text-grey">
                <p style={{ textAlign: 'right' }}>{customStrategy.name}</p>
              </div>
            ) : (
              <div className="is-flex" onClick={openStrategyModal}>
                <div className="cursor-pointer has-text-grey">
                  {strategyName}
                </div>
              </div>
            )}
          </>
        }

```

Reply:
also updating this component: https://github.com/DapperCollectives/CAST/blob/7b11685bdebd3d82dbf4dab93247a8235e530018/frontend/packages/client/src/components/ProposalInformation/InfoBlock.js#L19-L20
to remove the height:
```
<div
        className="column p-0 is-flex flex-1 is-align-items-center is-justify-content-flex-end"
        style={content ? { height: '1.5rem' } : {}}
      >

Reply:
Updated.